drafting/conda-template.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+# This script reads in a yaml file (qiime2 recipe)
+# Pulls the needed data from that file
+# And uses that data to write to a secondary yaml file
+# Which will be the end result: a conda recipe
+
+import yaml
+import os
+import sys
+from jinja2 import Template
+
+if __name__ == '__main__':
```

Comment:
This is looking great! Now let's take some time to refactor this into smaller functions.

Some possible function ideas:

- argv validation all in one place
- input file loading/parsing
- output file rendering/writing

Check out the scripts here for some additional ideas:

https://github.com/qiime2/library/tree/master/bin

Reply:
@thermokarst thank you! Just pushed up a refactor that breaks this up into three smaller functions (per your recommendations above). Let me know what you think!